server/plugin/api_test.go

```diff
@@ -188,10 +190,11 @@ func TestHandleVote(t *testing.T) {
 				store.PollStore.On("Save", poll2Out).Return(nil)
 				return store
 			},
-			Request:            &model.PostActionIntegrationRequest{UserId: "userID1", PostId: "postID1"},
+			Request:            &model.PostActionIntegrationRequest{UserId: "userID1", ChannelId: "channelID1", PostId: "postID1"},
 			VoteIndex:          1,
 			ExpectedStatusCode: http.StatusOK,
-			ExpectedResponse:   &model.PostActionIntegrationResponse{EphemeralText: responseVoteUpdated.Other, Update: expectedPost2},
+			ExpectedMsg:        "Your vote has been updated.",
```

Comment:
I think that using `responseVoteCounted.Other` is more stable than using text. 2/5

Reply:
I'm been thinking about the same while rewriting the tests. I'm inclined to make tests more "stand alone". To me tests should not be written clever. They should be boring but easy to understand.

When I change something in the code I **want** to be forced to think about if this breaks a test. This is why I opted to use the explicit strings here.

Let me know what you think about this.

Reply:
Tha followings are my thoughts for using variables instead of message text itself.

* Stable for changes of message text
  * No need to modify any tests when text changes
  * Tests don't test what text is returned, but test which message is returned, because all messages declared as variables
* Easy to search places where that message is used
  * Message text itself is like a magic number, so it should be avoided.

However, I also agree with your thoughts. Additionally, even if a message text is changed for now, fixing tests would be easy step because of little number of tests related to that text.

So let's go with this for now. thank you for explaining your thoughts.